src/css/components/SelectBox.scss

```diff
@@ -68,11 +81,19 @@ $SelectBox-indicator-width: scaleGrid(4);
     outline: none;
   }
 
+  &:invalid {
+    color: red !important;
```

Comment:
Use `negative` color variable?

Reply:
oops... was testing something to get that to show, but wasn't getting it to work.. cutting